mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ActionButtons.tsx

```diff
@@ -110,13 +134,22 @@ function ActionButton(props: ActionButtonsProps): JSX.Element {
         onPress={props.onPress}
         testID={props.testID}
       >
-        <ThemedIcon
-          dark={tailwind("text-mono-dark-v2-900")}
-          light={tailwind("text-mono-light-v2-900")}
-          iconType={props.iconType}
-          name={props.icon}
-          size={props.iconSize ?? 24}
-        />
+        {props.iconType === undefined ? (
+          <DFIIcon
+            width={props.iconSize}
+            height={props.iconSize}
+            color={isLight ? "#000" : "#FFF"}
```

Comment:
```suggestion
            color={getColor(isLight ? "mono-light-v2-900" : "mono-dark-v2-900")}
```
can use getColor() instead of hardcoding it
also, could you check with the designers if the color is correct in figma? dark mode icon is using 00 but light icon is using 900